packages/client/hmi-client/src/services/project.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Project
+ */
+
+import API from '@/api/api';
+import { Project } from '@/types/Project';
+
+// Define an empty Project
+const emptyProject = {
+	id: '',
+	name: 'Empty project',
+	description: 'This is an empty project, used as placeholder',
+	timestamp: ''
+} as Project;
+
+/**
+ * Get a project per id
+ * @return Project - the appropriate project, or an empty one if none returned by API
+ */
+async function getProject(projectId: string): Promise<Project> {
+	const response = await API.get(`/projects/${projectId}`);
+	return response?.data ?? emptyProject;
+}
+
+export { getProject as get, emptyProject as empty };
```

Comment:
Why do we need to rename the exported functions? If you want to call it `get` and `empty`, why not just name them get/empty respectively?